src/collective/importexport/browser/import_view.py

```diff
@@ -258,16 +274,61 @@ def export_file(result, header_mapping, request=None):
                     continue
                 if value is field.missing_value:
                     continue
+                if IRichTextValue.providedBy(value):
+                    # Convert to plain text
+                    transforms = api.portal.get_tool('portal_transforms')
```

Comment:
Getting a tool takes some time, so it would be better if you move this out of the for loop, no need to get it multiple times.